docs/source/onedal/build_app/build-application.rst

```diff
@@ -31,7 +31,7 @@ Applications on Linux* OS
 
 #. Set environment variables by calling ``<install dir>/setvars.sh``.
 
-#. Build your application with dpcpp:
+#. Use the C++ driver with the -fsycl option to build the application:
```

Comment:
don't think that regular users would understand driver terminology - suggest to just call out executable as it was done previously.

Reply:
@napetrov I'm not sure what you mean by driver terminology. We cannot leave it as it is, since there will be no dcppp anymore. What do you think about: 

`` Build your application with the C++ driver: ``

Reply:
use of icpx /icx names instead of C++ driver. Or some other correct name describing this